app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  has_many :likes, as: :likeable
```

Comment:
Are we liking comments? :thinking: there is no such functionality in your view and it isn't really required except of course you want to add it

Reply:
Solved!

Reply:
You're totally right, that was our idea, but we got into the requirements and forgot that we were planning to do that :D. Thanks for bringing that up!